packages/compiler/core/program.ts

```diff
@@ -962,12 +979,14 @@ export async function createProgram(
   }
 }
 
+// (nicholg) Why do we have both compile and createProgram?
```

Comment:
todo? should we just remove `compile`?

Reply:
Yes, TODO. Didn't want to make a breaking change as part of this.

Reply:
I'll remove the comment and file an issue